From 622756d8b080cc269f5dec53e82c61662f56de81 Mon Sep 17 00:00:00 2001
From: ownia <o451686892@gmail.com>
Date: Wed, 24 Aug 2022 22:38:06 +0800
Subject: [PATCH] Fix perfbook macos build

[git url]: https://git.kernel.org/pub/scm/linux/kernel/git/paulmck/perfbook.git
[branch]: master
[HEAD commit id]: 641067ba70383ad751512bd02cfc7cec17647523
[Platform]: macOS

Signed-off-by: ownia <o451686892@gmail.com>
---
 Makefile              | 7 +------
 utilities/autodate.sh | 6 +++---
 2 files changed, 4 insertions(+), 9 deletions(-)

diff --git a/Makefile b/Makefile
index 6ee849fd..600920b8 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,3 @@
-SHELL = /bin/bash
-
 GITREFSTAGS := $(shell ls -d .git/refs/tags 2>/dev/null)
 
 LATEXSOURCES = \
@@ -413,10 +411,7 @@ endif
 $(EPSSOURCES_FROM_FIG): $(FIXANEPSFONTS) $(FIXFONTS)
 $(EPSSOURCES_FROM_FIG): %.eps: %.fig
 	@echo "$< --> $(suffix $@)"
-ifndef FIG2EPS
-	$(error $< --> $@: fig2eps not found. Please install fig2ps)
-endif
-	@fig2eps --nogv $< > /dev/null 2>&1
+	/Users/ownia/codespace/fig2ps-1.5/bin/fig2ps --nogv --eps $< > /dev/null 2>&1
 	@sh $(FIXANEPSFONTS) $@
 
 # .eps --> .pdf rules
diff --git a/utilities/autodate.sh b/utilities/autodate.sh
index e33c254d..cfe24cb3 100644
--- a/utilities/autodate.sh
+++ b/utilities/autodate.sh
@@ -91,9 +91,9 @@ else
 		;;
 	esac
 fi
-month=`date --date="$date_str" +%B`
-year=`date --date="$date_str" +%Y`
-day=`date --date="$date_str" +%e`
+month=`date -R +%B`
+year=`date -R +%Y`
+day=`date -R +%e`
 if test -n "$release"
 then
 	release=`env printf '%s %s' '\\\\' "$release"`
-- 
2.37.2

